TYPO3.Flow/Classes/TYPO3/Flow/Http/RequestHandler.php

```diff
@@ -111,8 +116,8 @@ public function handleRequest()
             $this->request->setBaseUri(new Uri($this->settings['http']['baseUri']));
         }
 
-        $componentContext = new ComponentContext($this->request, $this->response);
-        $this->baseComponentChain->handle($componentContext);
+        $this->componentContext = new ComponentContext($this->request, $this->response);
```

Comment:
Good point, I guess it even makes a lot of sense generally to make this unproxyable.